styles/style.css

```diff
@@ -9,12 +9,15 @@
 
 .img-partners {
     display:inline-block;  
-    margin:25px;
+    /*margin:25px;*/
```

Comment:
How would we make this change so that it doesn't affect desktop and only applies to mobile?

Reply:
it looks like this change removes some of the whitespace that was at the bottom of the logo area:

![image](https://user-images.githubusercontent.com/2359538/35189523-a9963ec4-fe11-11e7-8485-02fc10e6f289.png)